tools/osbuild-dev

```diff
@@ -0,0 +1,209 @@
+#!/usr/bin/env python3
+"""`osbuild-dev` provides helper functionality for `osbuild` development
+mostly centered around manifest reading."""
+
+# pylint: disable=unsupported-membership-test,unsupported-delete-operation
+# pylint: disable=unsubscriptable-object
+
+import json
+import os
+import secrets
+import subprocess
+import tempfile
+from typing import Any, Optional
+
+try:
+    import attrs
+    import rich
+    import typer
+    from rich.tree import Tree
+except ImportError as import_error:
+    print(
+        "You are missing dependencies, please install `python3-attrs`, `python3-rich`, `python3-typer` or their `pip` equivalents."
+    )
+    raise
+
+
+cli = typer.Typer()  # Main command.
+man = typer.Typer()  # Manifest subcommand.
+
+cli.add_typer(man, name="manifest")
+
+con = rich.console.Console()
+
+
+@cli.callback()
+def main() -> int:
+    return 0
+
+
+def json_as_terminal_tree(tree: Tree, data: dict[str, Any], name: str) -> Tree:
+    """Convert JSON into a `rich` tree."""
+
+    if tree is None:
+        tree = Tree("")
+
+    if isinstance(data, (int, str)):
+        subtree = tree.add(f"{name}: [bold]{data}[/bold]")
+    elif isinstance(data, dict):
+        subtree = tree.add(str(name))
+        for key, val in data.items():
+            json_as_terminal_tree(subtree, val, key)
+    elif isinstance(data, list):
+        name = f"{name} [italic]({len(data)})[/italic]"
+        subtree = tree.add(name)
+        for index, item in enumerate(data):
+            json_as_terminal_tree(subtree, item, index)
+    else:
```

Comment:
Argument-type is `dict`, this needs to be updated, doesn't it?

Reply:
Yea, it should be Any (or a very large union, fixeD).